resources/aws/securitygroup.go

```diff
@@ -0,0 +1,77 @@
+package aws
+
+import (
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/ec2"
+	microerror "github.com/giantswarm/microkit/error"
+)
+
+var (
+	portsToOpen []int = []int{
+		// SSH
+		22,
+		// Insecure apiserver
+		80,
+		// apiserver
+		443,
```

Comment:
Not all of them, but probably everything except SSH. I can use what is available in TPR.